test/match.js

```diff
@@ -8,12 +8,12 @@ describe('match', function() {
 
   it('determines whether a string matches a regex', function() {
     assert.strictEqual(R.match(re, 'B17-afn').length, 1);
-    assert.strictEqual(R.match(re, 'B1-afn'), null);
+    assert.deepEqual(R.match(re, 'B1-afn'), []);
```

Comment:
That sounds really weird to me. How can an assertion framework let that pass? I have checked around in the ramda test code and the `toString` is not used in any other case. To fix this I would suggest switch assertion framework. I use chai in my projects and it throws an assertion error in these cases.